docs/0.1.0/install_macos.md

```diff
@@ -2,128 +2,184 @@
 
 ## Prerequisites
 
-1. Access to a synched RSK node. You can do this in a variety of ways:
-	1. Run your own node on TestNet or MainNet, see [https://github.com/rsksmart/rskj/wiki/Install-RskJ-and-join-the-RSK-Orchid-Mainnet-Beta](https://github.com/rsksmart/rskj/wiki/Install-RskJ-and-join-the-RSK-Orchid-Mainnet-Beta)
-	2. Compile and run a RSK node locally, see [https://github.com/rsksmart/rskj/wiki/Compile-and-run-a-RSK-node-locally](https://github.com/rsksmart/rskj/wiki/Compile-and-run-a-RSK-node-locally)
-2. An RSK account with an RBTC balance NOT lower than 0.001 RBTC
-3. XCode
+1. Access to a synced RSK node. You can do this in a variety of ways:
+	1. Run your own node on Testnet or Mainnet, see [Node (RSKj): Install](https://developers.rsk.co/rsk/node/install/).
+	2. Compile and run a RSK node locally, see [Node (RSKj): Contribute](https://developers.rsk.co/rsk/node/contribute/).
+
+## Recommended
+
+1. An RSK account with an RBTC balance of **at least** 0.001 RBTC, in order to be able to use all of the system features.
 
 ## Install required libraries/software
 
-### Install command line tools
+### 1. Install command line tools
 
-```
+Do this by opening a terminal and executing:
+
+```shell script
 xcode-select --install
 ```
 
-### Install Homebrew
+### 2. Install Homebrew
 
-From a Terminal window run:
+You can install Homebrew by executing:
 
-```
-/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```shell script
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 ```
 
-### Install OpenSSL
+### 3. Install OpenSSL
 
-```
+Do this by executing:
+
+```shell script
 brew install openssl
 export LC_ALL="en_US.UTF-8"\nexport LC_CTYPE="en_US.UTF-8"
 export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
 ```
 
-### Install Python 3.7
+### 4. Install Python 3.7
 
+We'll need Python 3.7 specifically, so install it through:
 
-```
-brew install python3
+```shell script
+brew install python@3.7
 ```
 
-### Install PIP
+### 5. Install `pip`
 
-```
+The package installer for Python can be installed through:
+
+```shell script
 sudo easy_install pip
 ```
 
-### Install virtualenv
+### 6. Install `virtualenv`
 
+Install `virtualenv` by executing:
 
-
-```
+```shell script
 pip3 install virtualenv
 ```
 
-### Install libpq
+### 7. Install database tools
 
-```
+Execute these 2 commands:
+
+```shell script
 brew install libpq
 ```
 
-### Install postgresql
-
-```
+```shell script
 brew install postgresql
 ```
```

Comment:
relevant for these 2: [RLP-1018](https://jirainfuy.atlassian.net/browse/RLP-1018)